clang/include/clang/Driver/Options.td

```diff
@@ -8447,9 +8447,6 @@ def fsycl_use_main_file_name : Flag<["-"], "fsycl-use-main-file-name">,
   HelpText<"Tells compiler that -main-file-name contains an absolute path and "
            "file specified there should be used for checksum calculation.">,
   MarshallingInfoFlag<CodeGenOpts<"SYCLUseMainFileName">>;
-def fsycl_allow_virtual_functions : Flag<["-"], "fsycl-allow-virtual-functions">,
```

Comment:
is this technically an abi break because we are removing this option? or does it not count since this was an experimental option?

Reply:
This is not a clang driver option, but a `-cc1` option, i.e. it has to be used through `-Xclang`. Considering that the option was internal and undocumented, I think that we can drop it without any deprecation periods.

Reply:
It probably would have been clearer if this was marked with `Visibility<[CC1Option]>` which I  should have caught when this was added.

Reply:
no big deal i was just wondering :P